README.md

```diff
@@ -93,6 +95,10 @@ The CDP4ServicesDal library is a C# library that provides an implementation of E
 
 The CDP4WspDal library is a C# library that provides an implementation of ECSS-E-TM-10-25A Annex C.2, the JSON REST API. The CDP4WspDal can be used to communicate with any ECSS-E-TM-10-25A Annex C.2 implementation, including the COMET Services.
 
+## CDP4Web
+
+The CDP4Web library is a C# library that provides helpful classes that facilitate the CDP4ServicesDal to any client.
```

Comment:
The CDP4Web library is a C# library that provides helpful classes that facilitate the use of the  CDP4ServicesDal to any client